ABOUT.md

```diff
@@ -4,99 +4,118 @@ About nGraph Compiler stack
-future, we will further expand the capabilities of Hybrid transformer 
-by enabling more features, such as localized cost modeling and memory 
-sharing, when the next generation of NNP (Neural Network Processor) 
-is released. In the meantime, your deep learning software engineering 
-or modeling can be confidently built upon this stable anchorage.
-
-The Intel® Architecture IA (Intel® Architecture) transformer provides 
-two modes that reduce compilation time, and have already been shown 
-as useful for training, deploying, and retraining a deep learning 
-workload in production. For example, in our tests, DEX mode reduced 
-ResNet50 compilation time by 30X.
-
-We are excited to continue our work in enabling distributed training, 
-and we plan to expand the nodes to 256 in Q4 ‘18. Additionally, we 
+#### Bridge
+
+Starting from the top of the stack, nGraph receives a computational graph
+from a deep learning framework such as TensorFlow* or MXNet*. The
+computational graph is converted to an nGraph internal representation
+by a bridge created for the corresponding framework.
+
+An nGraph bridge examines the whole graph to pattern match subgraphs
+which nGraph knows how to execute, and these subgraphs are encapsulated.
+Parts of the graph that are not encapsulated will default to framework
+implementation when executed.
+
+#### nGraph Core
+
+nGraph uses a strongly-typed and platform-neutral
+`Intermediate Representation (IR)` to construct a "stateless"
+computational graph. Each node, or op, in the graph corresponds to
+one `step` in a computation, where each step produces zero or
+more tensor outputs from zero or more tensor inputs.
+
+This allows nGraph to apply its state of art optimizations instead
+of having to follow how a particular framework implements op execution,
+memory management, data layouts, etc.
+
+In addition, using nGraph IR allows faster optimization delivery
+for many of the supported frameworks. For example, if nGraph optimizes
+ResNet* for TensorFlow*, the same optimization can be readily applied
+to MXNet* or ONNX* implementations of ResNet*.
+
+#### Hybrid Transformer
+
+Hybrid transformer takes the nGraph IR, and partitions it into
+subgraphs, which can be assigned to the best-performing backend.
+There are two hardware backends shown in the stack diagram to demonstrate
+this graph partitioning. The Hybrid transformer assigns complex operations
+(subgraphs) to Intel® Nervana™ Neural Network Processor (NNP) to expedite the
+computation, and the remaining operations default to CPU. In the future,
+we will further expand the capabilities of Hybrid transformer
+by enabling more features, such as localized cost modeling and memory
+sharing.
+
+Once the subgraphs are assigned, the corresponding backend will
+execute the IR.
+
+
+#### Backends
+
+Focusing our attention on the CPU backend, when the IR is passed to
+the IA (Intel Architecture) transformer, it can be executed in two modes:
```

Comment:
The rule for acronyms is use the trademark first: Intel® Architecture (IA) then the abbreviation.